Sources/StreamChat/Database/DTOs/ChannelMemberListQueryDTO.swift

```diff
@@ -19,17 +19,20 @@ final class ChannelMemberListQueryDTO: NSManagedObject {
     @NSManaged var members: Set<MemberDTO>
         
     static func load(queryHash: String, context: NSManagedObjectContext) -> ChannelMemberListQueryDTO? {
-        let request = NSFetchRequest<ChannelMemberListQueryDTO>(entityName: ChannelMemberListQueryDTO.entityName)
-        request.predicate = NSPredicate(format: "queryHash == %@", queryHash)
-        return try? context.fetch(request).first
+        load(
+            keyPath: #keyPath(ChannelMemberListQueryDTO.queryHash),
+            equalTo: queryHash,
+            context: context
+        ).first
     }
     
     static func loadOrCreate(queryHash: String, context: NSManagedObjectContext) -> ChannelMemberListQueryDTO {
-        if let existing = Self.load(queryHash: queryHash, context: context) {
+        let request = fetchRequest(keyPath: "queryHash", equalTo: queryHash)
```

Comment:
Same as before. Shouldn't it use the method changed above?

Reply:
same as `AttachmentDTO`